.github/workflows/release-docker-image.yaml

```diff
@@ -49,6 +49,16 @@ on:
         description: Private key of a Github App being used for triggering other workflows
         required: true
 
+      sentry-auth-token:
+        description: Sentry authentication token
+        required: false
+      sentry-org:
+        description: Sentry organization
+        required: false
+      sentry-project:
+        description: Sentry project
+        required: false
```

Comment:
Why are these secrets? 
Additionally I would add default values for both. 
- org: "neohelden"
-  project: <github-repo-name>

Reply:
Sentry proposes those as secrets...

Reply:
I think this is a copy&paste error in the sentry readme. It does not make sense, to hide these details in a secret.